Gems/Terrain/Code/Source/Components/TerrainWorldComponent.cpp

```diff
@@ -14,6 +14,8 @@
 #include <AzCore/Serialization/EditContext.h>
 #include <AzCore/Serialization/SerializeContext.h>
 #include <AzFramework/Terrain/TerrainDataRequestBus.h>
+#include <QApplication>
+#include <QMessageBox>
```

Comment:
All of this needs to move over into EditorTerrainWorldComponent.  Qt shouldn't be linked to runtime code.